### ISSUE TITLE
Resolve accessibility test failures in 5.0 and 3.1

### ISSUE THREAD

**Issue by mkArtakMSFT** [dotnet/aspnetcore#32365]
We have two color-contrast issues reported through the accessibility testing:
- [ ] [Bug 1162814: A11y_.NETCoreJuly2020_ASP.Net Core_React Register_ColorContrast: Edit fields and buttons present on register page fail in color contrast ratio when we focus on them.](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1162814)
- [ ] [Bug 1162783: A11y_.NETCoreJuly2020_ASP.Net Core_React Register_ColorContrast: Edit fields present on register page fail in color contrast ratio for the border.](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1162783)

**Comment by mkArtakMSFT** [comment]
Looks like there is mandate to patch this for 5.0 and 3.1, so closing this.
